src/modules/analysis/AnalysisViewComponent.js

```diff
@@ -58,6 +54,18 @@ const getTableRow = a => {
   );
 };
 
+const FieldGroup = ({ id, label, md, value }) => (
```

Comment:
I agree with Ingar. Choose a different name for this mini component. FormGroup (the name) IS a standard component in react-bootstrap and will shadow an import.